### DISCUSSION TITLE
Add screenshot or gif in readme to demonstrate integration in action

### DISCUSSION THREAD

**Discussion by GabLeRoux** [webbertakken/reviewer#37] [Ideas]
This might reduce the friction and have people go forward with integrating it with their project. Or maybe just a link to the pull-requests to see it in action? Could be interesting to pin point a specific case where it was actually helpful :)

https://github.com/webbertakken/reviewer/pulls

<img width="909" alt="ScreenShot Google Chrome 2023-07-16 at 12 22 43@2x" src="https://github.com/webbertakken/reviewer/assets/1264761/f9ce9a2b-29ac-4fc0-9401-486d3a78203d">

Might be interesting to mention caveats as well 😅 but I assume anyone using GPT already knows about this by now.

**Comment by webbertakken** [comment]
Thanks for trying Reviewer! <3

Definitely need to filter the messages where GPT is basically only responding with "my training data" and "cutoff" or "not connected to internet".

Great suggestion. Indeed, I'm looking to make a video / gif for the readme as well, hopefully once it's able to respond with 3 simple points and not the cutoff or not-connected message. Happy to accept a preliminary one for now if you have any nice ideas.
